migrations/utils/initSettings.js

```diff
@@ -40,4 +42,68 @@ module.exports = async function(
             txConfig,
         );
     }
+}
+
+/**
+ * We add initial Node Components version to Settings contract to start the platform.
+ * 
+ * @param settingsInstance a Settings contract instance. 
+ * @param web3 web3 library reference.
+ */
+function initNodeComponents(settingsInstance, nodeComponentsVersions, web3) {
+    const WEB2 = 'WEB2';
```

Comment:
I would create a JS file with the constants (maybe test/utils/componentNames.js). Similar to test/utils/chains.js

Reply:
Extracted constants into a newly created constants file with components' names.